source/query/queries.txt

```diff
@@ -6,6 +6,14 @@ View Past Queries
 
 .. default-domain:: mongodb
 
+Starting in |compass| 1.31, you can view your favorite and saved 
+queries and aggregation pipelines on the :guilabel:`My Queries` view 
+once connected to your cluster.
```

Comment:
Can we explain what happens when a user clicks a query or aggregation?

Reply:
Added two small blurbs to address this.